assets/js/modules/idea-hub/components/setup/Setup.stories.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Idea Hub setup component stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from '../../datastore/constants';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+import SetupMain from './SetupMain';
+
+const features = [ 'ideaHubModule' ];
+
+function Template() {
+	return (
+		<div className="googlesitekit-setup">
+			<section className="googlesitekit-setup__wrapper">
+				<div className="mdc-layout-grid">
+					<div className="mdc-layout-grid__inner">
+						<div className=" mdc-layout-grid__cell mdc-layout-grid__cell--span-12 ">
+							<SetupMain />
+						</div>
+					</div>
+				</div>
+			</section>
+		</div>
+	);
+}
+
+export const DefaultSetup = Template.bind( null );
+DefaultSetup.storyName = 'Set up form';
+DefaultSetup.parameters = { features };
+DefaultSetup.decorators = [
+	( Story ) => {
+		const moduleFixture = [ {
+			slug: 'idea-hub',
+			name: 'Idea Hub',
+			description: "Idea Hub suggests what you can write about next, based on searches that haven't been answered yet",
```

Comment:
At first glance, I thought: *I think elsewhere we tend to use gettext/`__()` functions for strings like these where they'd normally be translated*.

But then I thought "okay, this is a fixture for an API response which would always return a string and be translated/have the locale handled in PHP".

So I guess I'm just noting that thought here for future reference—this is correct, but it's sort of a subtle difference because many stories use gettext, even if we can't really view translated stories right now I think.

`</matt-thoughts>` 😅

Reply:
I went round a similar mental merrygoround 😄  🎠
Honestly I didn't know what the protocol was for things like this where, as you say, the text is coming pre-translated from the back end.